packages/babel-helpers/src/helpers/applyDecs2311.ts

```diff
@@ -469,32 +467,22 @@ export default /* @no-mangle */ function applyDecs2311(
   }
 
   /* @no-mangle */
-  function applyMemberDecs(
-    Class: any,
-    decInfos: DecoratorInfo[],
-    instanceBrand: Function,
-    metadata: any,
-  ) {
+  function applyMemberDecs() {
     var ret: Function[] = [];
     var protoInitializers: Function[];
     var staticInitializers: Function[];
     var staticBrand = function (_: any) {
-      return checkInRHS(_) === Class;
+      return checkInRHS(_) === targetClass;
     };
 
-    var existingNonFields = new Map();
-
     function pushInitializers(initializers: Function[]) {
       if (initializers) {
         ret.push(runInitializers.bind(null, initializers));
       }
     }
 
-    for (var i = 0; i < decInfos.length; i++) {
-      var decInfo = decInfos[i];
-
-      // skip computed property names
-      if (!Array.isArray(decInfo)) continue;
```

Comment:
I'm not sure if this is useful, there are no tests related to it.

Reply:
Yeah `decInfo` is always an array. It's probably a leftover from an old version.